internal/handlers/images_test.go

```diff
@@ -68,4 +68,31 @@ var _ = Describe("ServeHTTP", func() {
 		Expect(err).NotTo(HaveOccurred())
 		Expect(respContent).To(Equal([]byte("initrdcontent")))
 	})
+
+	It("calls the initrdAddrsize handler ServeHTTP", func() {
+		imageID := "bf25292a-dddd-49dc-ab9c-3fb4c1f07071"
+		stubISOHandler := http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+			Fail("ISO handler should not have been called")
+		})
+		stubInitrdAddrsize := http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+			Expect(r.Method).To(Equal(http.MethodGet))
+			Expect(r.URL.Path).To(Equal(fmt.Sprintf("/images/%s/initrd-addrsize", imageID)))
+			http.ServeContent(w, r, "initrd.addrsize", time.Now(), strings.NewReader("initrdcontent"))
+		})
+
+		imageHandler := &ImageHandler{
+			long:           stubISOHandler,
+			initrdAddrsize: stubInitrdAddrsize,
+		}
```

Comment:
Would this work as well?

```suggestion
		stubInitrdAddrsize := http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
			Expect(r.Method).To(Equal(http.MethodGet))
			Expect(r.URL.Path).To(Equal(fmt.Sprintf("/images/%s/initrd-addrsize", imageID)))
			http.ServeContent(w, r, "initrd.addrsize", time.Now(), strings.NewReader("initrdcontent"))
		})

		imageHandler := &ImageHandler{
			initrdAddrsize: stubInitrdAddrsize,
		}
```